aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Orchestrator.java

```diff
@@ -94,32 +97,88 @@ public Orchestrator(
         this.storageObserver = new StorageObserver(localStorageAdapter, mutationOutbox);
     }
 
+    /**
+     * Checks whether the orchestrator is {@link OrchestratorStatus#STARTED}.
+     * @return True if the orchestrator is started, false otherwise.
+     */
+    public boolean isStarted() {
+        return OrchestratorStatus.STARTED.equals(status.get());
+    }
+
     /**
      * Start performing sync operations between the local storage adapter
      * and the remote GraphQL endpoint.
      * @return A Completable operation to start the sync engine orchestrator
      */
     @NonNull
     public Completable start() {
-        return mutationOutbox.load()
-            .andThen(Completable.fromAction(() -> {
-                storageObserver.startObservingStorageChanges();
-                subscriptionProcessor.startSubscriptions();
-                syncProcessor.hydrate().blockingAwait();
-                mutationProcessor.startDrainingMutationOutbox();
-                subscriptionProcessor.startDrainingMutationBuffer();
-                LOG.info("Cloud synchronization is now fully active.");
-            }));
+        // Only start if it's stopped.
+        if (OrchestratorStatus.STOPPED.equals(status.get())) {
```

Comment:
To help with the indentation / nesting, can we invert this?

```
if (!OrchestratorStatus.STOPPED.equals(status.get())) {
    return initializationCompletable;
}
// Save 4 cols, here ...
```

You can save another eight more cols with the `andThen(` on L120, too, by:
```
initializationCompletable = mutationOutbox.load().andThen(Completable.fromAction(() -> {
    // 8 left of where it was 
}
```

Reply:
Heck yeah...that's much better.